packages/devextreme/testing/tests/DevExpress.ui.widgets/multiView.tests.js

```diff
@@ -124,6 +124,41 @@ QUnit.module('rendering', () => {
         }
     });
 
+    QUnit.test('multiview items.visible option should always be default to true', function(assert) {
+        const $multiView = $('#multiView').dxMultiView({
+            items: [
+                { text: '1' }
+            ]
+        });
+        const $items = $multiView.find(toSelector(MULTIVIEW_ITEM_CLASS));
+
+        assert.notOk($items.eq(0).hasClass(MULTIVIEW_ITEM_HIDDEN_CLASS), 'first item is visible');
+    });
+
+    QUnit.test('multiview should not render items those that are option visible is false', function(assert) {
+        const $multiView = $('#multiView').dxMultiView({
+            items: [
+                { text: '1', visible: true },
+                { text: '2', visible: false }
+            ]
+        });
+        const $items = $multiView.find(toSelector(MULTIVIEW_ITEM_CLASS));
+
+        assert.notOk($items.eq(0).hasClass(MULTIVIEW_ITEM_HIDDEN_CLASS), 'first item is visible');
+        assert.ok($items.eq(1).hasClass(MULTIVIEW_ITEM_HIDDEN_CLASS), 'second item is hidden');
+    });
+
+    QUnit.test('when all items are not visible selectIndex should be 0', function(assert) {
+        const multiView = $('#multiView').dxMultiView({
+            items: [
+                { text: '1', visible: false },
+                { text: '2', visible: false }
+            ]
+        }).dxMultiView('instance');
+
+        assert.strictEqual(multiView.option('selectedIndex'), 0, 'selectedIndex is back at 0');
+    });
```

Comment:
[excess test]
I think it's better to remove this tests for now, it checks nothing because default value of a `selectedIndex` is `0`. 
But if u set `selectedIndex: 1` in initial config, it'll not work

Reply:
[eb9495e](https://github.com/DevExpress/DevExtreme/pull/28128/commits/eb9495e5a1192db390e67834c0ed4c2764c2b9d4)